bodhi/server/scripts/approve_testing.py

```diff
@@ -96,19 +95,8 @@ def main(argv=sys.argv):
                     config.get('testing_approval_msg') % update.release.mandatory_days_in_testing)
                 update.comment(db, text, author=u'bodhi')
 
-
-def _get_db_session(config_uri):
-    """
-    Construct and return a database session using settings from the given config_uri.
-
-    :param config_uri: A path to a config file to use to get the db settings.
-    :type  config_uri: basestring
-    :return:           A database session
-    """
-    # There are many blocks of code like this in the codebase. We should consolidate them into a
-    # single utility function as described in https://github.com/fedora-infra/bodhi/issues/1028
-    settings = get_appsettings(config_uri)
-    engine = engine_from_config(settings, 'sqlalchemy.')
-    Session = scoped_session(sessionmaker(extension=ZopeTransactionExtension()))
-    Session.configure(bind=engine)
-    return Session()
+        db.commit()
+    except SQLAlchemyError as e:
```

Comment:
What if some other exception happens here? Lke what if the string formatting fails above and raises an Exception? I assume that postgres will automatically roll a transaction back if the connection is lost from the process exiting?